Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -473,6 +474,7 @@ void MenuBar::UpdateToolsMenu(bool emulation_started)
   m_pal_ipl->setEnabled(!emulation_started &&
                         File::Exists(SConfig::GetInstance().GetBootROMPath(EUR_DIR)));
   m_import_backup->setEnabled(!emulation_started);
+  m_check_nand->setEnabled(!emulation_started);
```

Comment:
Does this make sense when there is no NAND at all? But then again, this probably goes for the other menu items there too (like extract certificates etc.)

Reply:
In my opinion, yes. Users that only use games can still end up corrupting the NAND (thanks to Dolphin :P), and the NAND stays broken even after making it more complete (by installing system titles).